### ISSUE TITLE
Vue 3

### ISSUE THREAD

**Issue by luanid** [fusioncharts/vue-fusioncharts#83]
Is there any plan to support vue 3?

**Comment by AyanBhadury** [comment]
Yes absolutely.
@sikrigagan @niharsawant  FYI

**Comment by martinmalia** [comment]
What is the timeline on this?

**Comment by ElectroLutz** [comment]
Any news?

**Comment by Fbartoli** [comment]
Bump ?

**Comment by ElectroLutz** [comment]
Any chance this will ever happen?

**Comment by AyanBhadury** [comment]
Yes the request is under development will be done in the upcoming releases

**Comment by ElectroLutz** [comment]
@AyanBhadury it's been 3 months. Any news? Or maybe a release date?

**Comment by AyanBhadury** [comment]
@ElectroLutz Will be released in the next version release, which is expected to be done by this quarter.

**Comment by AyanBhadury** [comment]
**Vue3 support is added please upgrade to version 3.2.0**: https://www.npmjs.com/package/vue-fusioncharts

Demo: https://github.com/AyanBhadury/vue-fc-3

@ElectroLutz @luanid @Fbartoli
